File: source/kCura.Relativity.Export.ConvertedToC#/Exporters/LineFactory/LineFactoryBase.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Data;
using System.Linq;
namespace kCura.Relativity.Export.Exports.LineFactory
{
	public abstract class LineFactoryBase
	{
		public abstract void WriteLine(System.IO.StreamWriter stream);

		protected LineFactoryBase()
		{
			//Satifies Rule: Abstract types should not have constructors
		}
	}
}

File: source/kCura.Relativity.Export/Exporters/LineFactory/LineFactoryBase.vb
Namespace kCura.Relativity.Export.Exports.LineFactory
	Public MustInherit Class LineFactoryBase
		Public MustOverride Sub WriteLine(ByVal stream As System.IO.StreamWriter)

		Protected Sub New()
			'Satifies Rule: Abstract types should not have constructors
		End Sub
	End Class
End Namespace
